TOC.md

```diff
@@ -191,9 +194,9 @@
     - [TiDB Controller](dev/reference/tools/tidb-control.md)
     - [工具下载](dev/reference/tools/download.md)
 + 常见问题 (FAQ)
-  - [TiDB FAQ](FAQ.md)
-  - [TiDB-Lightning FAQ](tools/lightning/faq.md)
-  - [升级 FAQ](op-guide/upgrade-faq.md)
+  - [TiDB FAQ](faq/tidb.md)
```

Comment:
This renaming affects the following links:

- Two links in [this file](https://github.com/pingcap/docs-cn/blob/master/dev/reference/error-codes.md)
- One link in [this file](https://github.com/pingcap/docs-cn/blob/master/dev/reference/tools/tidb-lightning/deployment.md)

Please update them.